src/main/java/com/google/api/generator/gapic/composer/common/AbstractTransportServiceStubClassComposer.java

```diff
@@ -200,11 +202,11 @@ public GapicClass generate(GapicContext context, Service service) {
             .setAnnotations(createClassAnnotations(service))
             .setScope(ScopeNode.PUBLIC)
             .setName(className)
-            .setExtendsType(
-                typeStore.get(getTransportContext().classNames().getServiceStubClassName(service)))
+            .setExtendsType(typeStore.get(ClassNames.getServiceStubClassName(service)))
```

Comment:
This does not look correct. The expectation is to move the other direction - convert static usages of class names to the dynamic ones.